src/ai/susi/server/api/cms/CreatePrivateSkillService.java

```diff
@@ -0,0 +1,209 @@
+package ai.susi.server.api.cms;
```

Comment:
up to here it's fine but I don't see a requirement to make a new servlet. The private skill shall also be made with the CreateSkillService.java servlet and that servlet needs to read a 'private' attribute and then act acordingly

Reply:
@Orbiter I have now removed the new API and made changes to the  `CreateSkillService.java` file to detect `private` attribute and then store them in `susi_private_skill_data` repository. Please review. Thanks,